<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CMakeBuildSettings</class>
 <widget class="QWidget" name="CMakeBuildSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>605</width>
    <height>607</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <layout class="QVBoxLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout">
     <item>
      <widget class="KComboBox" name="buildDirs"/>
     </item>
     <item>
      <widget class="QPushButton" name="addBuildDir">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Add</string>
       </property>
       <property name="icon">
        <iconset theme="list-add"/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="removeBuildDir">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Remove</string>
       </property>
       <property name="icon">
        <iconset theme="list-remove"/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Cache Values</string>
     </property>
     <layout class="QVBoxLayout">
      <item>
       <widget class="QTableView" name="cacheList"/>
      </item>
      <item>
       <widget class="QLabel" name="label_6">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Comment:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="commentText">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="showAdvanced">
     <property name="text">
      <string>Show Advanced</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="advancedBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Advanced</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::ExpandingFieldsGrow</enum>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="installationPrefixLabel">
        <property name="text">
         <string>Installation prefix:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="KUrlRequester" name="installationPrefix"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="buildTypeLabel">
        <property name="text">
         <string>Build type:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="buildType">
        <property name="editable">
         <bool>true</bool>
        </property>
        <item>
         <property name="text">
          <string notr="true">Debug</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string notr="true">Release</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string notr="true">RelWithDebInfo</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string notr="true">MinSizeRel</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="extraArgumentsLabel">
        <property name="text">
         <string>Extra arguments:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="KComboBox" name="extraArguments">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="editable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Configure environment:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="KDevelop::EnvironmentSelectionWidget" name="environment">
          <property name="sizePolicy">
           <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="toolTip">
           <string>Select an environment to be used</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="KDevelop::EnvironmentConfigureButton" name="configureEnvironment"/>
        </item>
       </layout>
      </item>
      <item row="5" column="1">
       <widget class="QCheckBox" name="showInternal">
        <property name="text">
         <string>Show advanced values</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="KUrlRequester" name="cMakeExecutable"/>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>CMake executable:</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>KUrlRequester</class>
   <extends>QWidget</extends>
   <header>kurlrequester.h</header>
  </customwidget>
  <customwidget>
   <class>KDevelop::EnvironmentSelectionWidget</class>
   <extends>KComboBox</extends>
   <header>util/environmentselectionwidget.h</header>
  </customwidget>
  <customwidget>
   <class>KDevelop::EnvironmentConfigureButton</class>
   <extends>QToolButton</extends>
   <header>shell/environmentconfigurebutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
